Dockerfile

```diff
@@ -16,12 +16,12 @@ RUN set -x && \
     # Clean-up
     apk del .deps
 
+ARG DOCKER_COMPOSE_VERSION
+ENV DOCKER_COMPOSE_VERSION=${DOCKER_COMPOSE_VERSION:-"1.10.0"}
+
 RUN set -x && \
     apk add --no-cache -t .deps ca-certificates curl && \
-    # Install docker-compose
```

Comment:
Please keep the comments, they help to understand and get latest version faster.